src/output/dnssim.lua

```diff
@@ -68,6 +69,45 @@ function DnsSim.new(max_clients)
     return setmetatable(self, { __index = DnsSim })
 end
 
+local function _check_version(version, req_version)
+    if req_version == nil then
+        return version
+    end
+    local min_version = tonumber(req_version)
+    if min_version == nil then
+        C.output_dnssim_log():critical("invalid version number: "..req_version)
```

Comment:
Why a critical here?

If you pass a invalid number it should be fatal right?

Because there is no way to check the return in a way that says the input was invalid.

Reply:
fixed